src/main/java/step3/StringUtil.java

```diff
@@ -0,0 +1,14 @@
+package step3;
+
+public class StringUtil {
+
+    public static String printToHyphen(int printCount) {
```

Comment:
이렇게 바꿔보는건 어떨까요? 어떤 장점이 있을까요?
```suggestion
public static String repeat(String string, int count) {
```

Reply:
이렇게 하니까 출력할 문자에 대해 유연하게 대처할 수 있군요!